src/components/GameGrid/Rounds/GroupStages.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+
+import GameRound from './GameRound'
+
+import { makeStyles } from '@material-ui/styles'
+
+import SinglePoll from '../../../models/SinglePoll'
+
+const useStyles = makeStyles({
+  groupStagesContainer: {
+    display: 'flex',
+    flexWrap: 'wrap',
+    gap: 8,
+    margin: 'auto',
+    alignItems: 'center',
+    justifyContent: 'center',
+  },
+  gameContainer: {
+    marginTop: 8,
+    width: 500,
+    maxWidth: '100%',
+  },
+})
+
+type GroupStageData = Record<number, SinglePoll>
+
+interface Props {
+  groupStageData: GroupStageData
+}
+
+const GroupStages: React.FC<Props> = ({ groupStageData }) => {
+  const classes = useStyles()
+
+  if (!groupStageData) return null
+
+  return (
+    <>
+      <section className={classes.groupStagesContainer}>
+        {Object.keys(groupStageData).map(key => (
+          <GameStage data={groupStageData[key]} key={key} />
```

Comment:
Forgot to mention this - just a minor misspell.
```suggestion
          <GameRound data={groupStageData[key]} key={key} />
```